### ISSUE TITLE
Use libzim IndexData::getContent to provide currated content to index.

### ISSUE THREAD

**Issue by mgautierfr** [openzim/sotoki#282]
libzim provides a way for scrappers to provide a different content than the one stored for the indexation.

It allow a better indexation when a lot of content is not relevant about the subject of the content itself.

mwoffliner should parse the html content and extract only the relevant information (so remove thing such has menu, footer, user information, links to other questions...)

See comments in https://github.com/openzim/libzim/issues/653

**Comment by kelson42** [comment]
@rgaudin @benoit74 Would this approach, here, brings a real improvement? If "yes', which one?

**Comment by rgaudin** [comment]
Improvement would be marginal I think because we don't include much non-content text in the HTML.
A side effect would be parsing all our output using an in-scraper HTML parser versus letting libzim do it.

**Comment by kelson42** [comment]
Agree, Without a clear problem to fix, I propose to close this issue.
